src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,112 @@
+        <ResourceDictionary>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Default">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <StackPanel Orientation="Vertical" Background="Black">
+        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
+            <Button x:Name="CloseButton" Padding="0" Click="_CloseClick" TabIndex="1">
+                <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE711;" FontSize="15" Foreground="White"/>
+            </Button>
+        </StackPanel>
+
+        <StackPanel x:Name="SuggestBoxStackPanel" Height="35" Orientation="Horizontal">
+            <TextBox  x:Name="TextBox" PlaceholderText="Search"
+                KeyDown = "QuerySubmitted"
+                Width="160" Margin="0,0,1,0" HorizontalAlignment="Left" VerticalAlignment="Center" TabIndex="2">
+            </TextBox>
+
+            <ToggleButton x:Name="SetGoBackwardButton" HorizontalAlignment="Right" Padding="1" TabIndex="3"

```

Comment:
Since most of the controls are just icons, there is no information provided to UIA for Accessibility tools about what the controls are.  To fix that, you should add AutomationProperties.Name="...' to these controls.

Reply:
added